OrbitClientGgp/ClientGgp.cpp

```diff
@@ -65,7 +65,8 @@ bool ClientGgp::RequestStartCapture(ThreadPool* thread_pool) {
   // TODO(kuebler): right now selected_functions is only an empty placeholder,
   //  it needs to be filled separately in each client and then passed.
   absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionInfo> selected_functions;
-  Capture::capture_data_ = CaptureData(pid, target_process_->GetName(), selected_functions);
+  Capture::capture_data_ =
+      CaptureData(pid, target_process_->GetName(), target_process_, selected_functions);
```

Comment:
Could we pass in only the target_process_ and get the name in the constructor? Or does it make sense to have the ability to specify a different name?

Reply:
As we are working towards removing OrbitProcess completely, I wanted to avoid this. But anyways, yes I can just pass this, if you wish to.